pkg/ranger/grpc_test.go

```diff
@@ -65,15 +65,17 @@ func TestGRPCRanger(t *testing.T) {
 		gomock.InOrder(calls...)
 
 		c := client.NewCustomRoute(context.Background(), route)
-		r, err := GRPCRanger(c, "")
+		rr, err := GRPCRanger(c, "")
 		if assert.NoError(t, err, errTag) {
-			assert.Equal(t, tt.size, r.Size(), errTag)
+			assert.Equal(t, tt.size, rr.Size(), errTag)
 		}
-		data, err := ioutil.ReadAll(r.Range(tt.offset, tt.length))
+		r, err := rr.Range(tt.offset, tt.length)
 		if tt.errString != "" {
 			assert.EqualError(t, err, tt.errString, errTag)
 			return
 		}
+		assert.NoError(t, err, errTag)
+		data, err := ioutil.ReadAll(r)
```

Comment:
tt.errString used to be checking the err from ioutil.ReadAll, and now it's only checking the err from rr.Range? I think something went weird here.